public/js/filters.js

```diff
@@ -56,4 +56,15 @@ angular.module('projectMigrationFilters', []).filter('getExtension', function()
       'box':'Migration to Box'
     }[String(type)] || 'Unknown Migration Type');
   };
+}).filter('readableExportTypeSmall', function() {
+  return function(type) {
+    return ({
+      'google': 'to Google Groups',
```

Comment:
Do we need to have google groups reference as we are migrating email using google groups